.travis.yml

```diff
@@ -27,7 +27,7 @@ script:
 - mvn clean install -Pjacoco -Pdocker coveralls:report
 - mvn clean verify -f saga-demo -Pdemo -Pdocker -Ddocker.useColor=false -Ddocker.showLogs
 after_success:
-- mvn deploy -DskipTests --settings .travis.settings.xml
+- if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then mvn deploy -DskipTests --settings .travis.settings.xml; fi
```

Comment:
how about checking if branch is master instead?

Reply:
see http://docs.travis-ci.com/user/ci-environment/
TRAVIS_BRANCH
* for push builds, or builds not triggered by a pull request, this is the name of the branch.
* for builds triggered by a pull request this is the name of the branch targeted by the pull request.
* for builds triggered by a tag, this is the same as the name of the tag (TRAVIS_TAG).

So it looks like in many case of the pull request, the branch target is "master".